cluster/gce/manifests/cloud-controller-manager.manifest

```diff
@@ -23,16 +23,14 @@
 "containers":[
     {
     "name": "cloud-controller-manager",
-    "image": "gcr.io/k8s-staging-cloud-provider-gcp/cloud-controller-manager:v1.25.2-alpha_ae91c1fc0c443c464a4c878ffa2a4544483c6d1f",
+    "image": "gcr.io/k8s-staging-cloud-provider-gcp/cloud-controller-manager:v26.2.4",
     "resources": {
       "requests": {
         "cpu": "{{cpurequest}}"
       }
     },
     "command": ["/cloud-controller-manager"],
     "args": [
-      "--log-file=/var/log/cloud-controller-manager.log",
```

Comment:
Removing temporarily to see if CI will go green, opened https://github.com/kubernetes/cloud-provider-gcp/issues/520 to follow-up

Reply:
this comment is stale, referenced issue explain the problem, cloud-provider-gcp moved to go-runner

Reply:
thanks!